x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/LogicalPlanOptimizerTests.java

```diff
@@ -7378,6 +7379,24 @@ public void testFunctionNamedParamsAsFunctionArgument() {
         assertEquals(DataType.DOUBLE, ee.dataType());
     }
 
+    public void testFunctionNamedParamsAsFunctionArgument1() {
```

Comment:
as a side note I don't see a difference between this test and the one we add to `AnalyzerTests`.
I guess we just follow an existing pattern - but as a follow up, we should discuss if both are needed.